db/migrations/0002_alter_moviesession_cinema_hall_and_more.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-11-12 19:25
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='moviesession',
+            name='cinema_hall',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions', to='db.cinemahall'),

```

Comment:
The `related_name` for the `cinema_hall` field has been changed to `movie_sessions`. Ensure that this change is reflected in all parts of your codebase where this relationship is used.